backend/protocol_rpc/endpoints.py

```diff
@@ -312,6 +271,70 @@ def clear_db_tables(db_client: DBClient, tables: list) -> dict:
     db_client.clear_tables(tables)
 
 
+def send_raw_transaction(
+    transactions_processor: TransactionsProcessor,
+    accounts_manager: AccountsManager,
+    signed_transaction: str,
+) -> dict:
+    # Decode transaction
+    decoded_transaction = decode_signed_transaction(signed_transaction)
+
+    # Validate transaction
+    if decoded_transaction is None:
+        raise InvalidTransactionError("Invalid transaction data")
+
+    from_address = decoded_transaction["from"]
+    if not accounts_manager.is_valid_address(from_address):
+        raise InvalidAddressError(
+            from_address, f"Invalid address from_address: {from_address}"
+        )
+
+    transaction_signature_valid = transaction_has_valid_signature(
+        signed_transaction, decoded_transaction
+    )
+    if not transaction_signature_valid:
+        raise InvalidTransactionError("Transaction signature verification failed")
+
+    to_address = decoded_transaction["to"]
+
+    transaction_data = {}
+    result = {}
+    transaction_type = -1
+    if to_address and to_address != "0x":
```

Comment:
Here we decide if the transaction is a contract deployment or a contract method calling.
The node should do this. Here, we should store the call data encoded in the transactions database.